Assets/Game/Addons/ModSupport/Editor/ModOrganizerSupport/ModOrganizerMenuItems.cs

```diff
@@ -16,11 +17,13 @@ internal class ModOrganizerMenuItems
         private const string FullMenu = ParentMenu + "/" + SubMenu + "/";
 
         [MenuItem(FullMenu + "Locate Data")]
-        static void Select()
+        static async void Select()
         {
             Debug.Log($"<color=orange>Select your Mod Organizer data directory.</color> " +
                       $"This path should contain your `profiles` and `mods` subfolders.");
 
+            await Task.Delay(500);
+
             string dataPath = EditorUtility.OpenFolderPanel("Select Mod Organizer data path",
                 String.Empty, String.Empty);
```

Comment:
Maybe the UX move would be not to mix console output with modal windows. If a window needs more context before being shown, then you should make the message a modal popup

Reply:
I'd normally agree but that's just an extra thing to click away once you learn what to do after the first time you do it. I agree this isn't terribly pretty but it seems the most comfortable to me. I'm open to ideas though